apps/console/src/features/groups/components/wizard/group-basics.tsx

```diff
@@ -86,35 +78,15 @@ export const GroupBasics: FunctionComponent<GroupBasicProps> = (props: GroupBasi
      * TODO : Discuss and add or remove the Hybrid domains
      *        to the dropdown.
      */
-    const groupDomains = [{
+    const groupDomains = [ {
         key: -1, text: SharedUserStoreConstants.PRIMARY_USER_STORE, value: SharedUserStoreConstants.PRIMARY_USER_STORE
```

Comment:
Possible max line issue here. If so better to format.

```suggestion
        key: -1,
        text: SharedUserStoreConstants.PRIMARY_USER_STORE,
        value: SharedUserStoreConstants.PRIMARY_USER_STORE
```

Reply:
Resolved by 97e62f1